
float32 distance
bool is_too_close
bool turtle1_near_bound
bool turtle2_near_bound
